### PR TITLE
Create a case class for storing article information

### PR DESCRIPTION
**Pull request by gaurav**
This PR incorporates the addition of an ArticleInfo case class from PR #13. It additionally tweaks the definition of ArticleInfo and modifies the triple generation of PubDate to use [dct:issued](http://purl.org/dc/terms/issued) and [xsd:gYear](https://www.w3.org/TR/xmlschema11-2/#gYear). It also adds a simple test to make sure the text extraction is working.

### CODE REVIEW

File: src/main/scala/org/renci/chemotext/Main.scala [hunk 1]
```diff
@@ -28,6 +28,34 @@ import akka.stream._
 import akka.stream.scaladsl._
 import io.scigraph.annotation.EntityAnnotation
 import io.scigraph.annotation.EntityFormatConfiguration
+import org.apache.jena.datatypes.xsd.XSDDatatype
+
+import scala.collection.{SortedSet, immutable}
+
+case class ArticleInfo(pmid:String,info:String,meshTermIDs:Set[String],years:Set[String])
```
Comment (balhoff) [inline src/main/scala/org/renci/chemotext/Main.scala]: I like to have spaces after commas. IntelliJ autoformat should do this automatically.

Comment (balhoff) [reply src/main/scala/org/renci/chemotext/Main.scala]: Are you asserting more than one year using the same predicate? Will that be weird?

**Review by balhoff — APPROVED** [review]
I think this is good, but see my question about multiple years.

File: src/main/scala/org/renci/chemotext/Main.scala (hunk 1 above)
Comment (gaurav) [reply src/main/scala/org/renci/chemotext/Main.scala]: Formatting fixed in 52b577f.

I was storing multiple years in each ArticleInfo object, but when converted into triples, each year would show up as a separate `:article dct:issued "year"` triple. However, I've modified ArticleInfo to have a single, optional year in 7823933, which will always be the first year reported by the XML library.

We oughtn't to have more than one PubDate in a particular entry, but I was being paranoid and making sure that no date was ignored. I think the right approach is to have separate fields for different kinds of dates, each of which can report multiple dates if that's what's in the XML, i.e. https://github.com/NCATS-Gamma/omnicorp/pull/18/files#diff-798b8cc22ef571f671245e15c411d9f8R65-R68

**Pull request merged**